### PR TITLE
Bump to v1.0.0-rc1

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by flouthoc** [comment]
@mheon There are two commits but first one overrides the second. Is this intentional ?

**Review by lsm5 — APPROVED** [review]
LGTM

**Comment by mheon** [comment]
@flouthoc Yes. The first commit is tagged as v1.0.0-rc1, and I'll push the tag once merged. The second bumps us back to `-dev` for continued development work.

File: test/001-basic.bats [hunk 1]
```diff
@@ -7,7 +7,7 @@ load helpers
 
 @test "netavark version" {
     run_netavark --version
-    assert "netavark 0.0.1" "expected version"
+    assert "netavark 1.0.0-dev" "expected version"
```
Comment (Luap99) [inline test/001-basic.bats:10]: There is another version check below which has to be changed

**Pull request merged**